k8s.gcr.io/images/k8s-staging-kueue/images.yaml

```diff
@@ -0,0 +1 @@
+# TODO: add images
```

Comment:
Can you add an empty map instead?

```
- name: kueue
  dmap:
```

Reply:
sure why not, I am not sure if the lint like that, let me check

Reply:
lint is happy with that being empty, so we are good to go

Reply:
You should not leave an empty map. The promoter expect a non-empty map. Please only the comment.

Reply:
Done

Reply:
thanks for the review @ameukam